packages/aws-cdk-lib/aws-ec2/lib/security-group.ts

```diff
@@ -843,4 +862,32 @@ interface SecurityGroupLookupOptions {
    * @default Don't filter on VPC
    */
   readonly vpc?: IVpc;
+
+  /**
+   * Security group description
+   *
+   * @default Don't filter on description
```

Comment:
`@default - Don't filter on description` (needs a hyphen)

Reply:
The existing params didn't include a hyphen (securityGroupName and securityGroupId). I can add it in to those as well.